src/modules/Contrast/Module.js

```diff
@@ -15,9 +15,27 @@ module.exports = function Contrast(options, UI) {
 
     var step = this;
 
-    function extraManipulation(pixels) {
-      pixels = require('./Contrast')(pixels, options.contrast);
-      return pixels;
+    let contrast = options.contrast;
+
+    contrast = Number(contrast);
+    if (contrast < -100) contrast = -100;
+    if (contrast > 100) contrast = 100;
+    contrast = (100.0 + contrast) / 100.0;
+    contrast *= contrast;
+
+    function changeContrast(p){
+      p -= 0.5;
+      p *= contrast;
+      p += 0.5;
+      p *= 255;
+      if (p < 0) p = 0;
```

Comment:
How about using `Math.min(0, p)` instead of the if stmt?

Reply:
Sure